cmd/edgectl/aes_install_errors.go

```diff
@@ -93,103 +100,30 @@ func (i *Installer) GetVersionsError(err error) Result {
 	}
 }
 
-// Unable to fetch the AES CRD manifests (aes-crds.yaml)
-func (i *Installer) AESCRDManifestsError(err error) Result {
+// Unable to download the latest Chart
+func (i *Installer) DownloadError(err error) Result {
 	i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
 
 	url := "https://www.getambassador.io/docs/latest/topics/install/help/aes-crd-manifests"
 
 	return Result{
-		ShortMessage: "The installer failed to download the AES CRD manifests",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES CRD manifests to continue installing Ambassador Edge Stack.", url),
-		URL:          url,
-		Err:          errors.Wrap(err, "Failed to download AES CRD manifests"),
-	}
-}
-
-// Unable to fetch the AES manifests (aes.yaml)
-func (i *Installer) AESManifestsError(err error) Result {
-	i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-
-	url := "https://www.getambassador.io/docs/latest/topics/install/help/aes-manifests"
-	return Result{
-		ShortMessage: "The installer failed to download the AES manifests",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES manifests to continue installing Ambassador Edge Stack.", url),
+		ShortMessage: "The installer failed to download the AES Chart",
+		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES Chart to continue installing Ambassador Edge Stack.", url),
 		URL:          url,
-		Err:          errors.Wrap(err, "Failed to download AES manifests"),
-	}
-}
-
-// Unable to parse the downloaded AES manifests
-func (i *Installer) ManifestParsingError(err error, matches []string) Result {
-	url := "https://www.getambassador.io/docs/latest/topics/install/help/manifest-parsing"
-
-	i.log.Printf("matches is %+v", matches)
-
-	return Result{
-		Report:       "fail_bad_manifests",
-		ShortMessage: "The installer failed to parse AES manifests",
-		Message:      fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about downloading AES manifests to continue installing Ambassador Edge Stack.", url),
-		URL:          url,
-		Err:          err,
+		Err:          errors.Wrap(err, "Failed to download AES CRD Chart"),
```

Comment:
"AES CRD Chart" -> "AES Chart" (for consistency)?

Reply:
Or perhaps event better "AES Chart" -> "Helm chart" (in both places here).

Reply:
I think we can reuse the error message by just saying `"Failed to download AES CRD"`

Reply:
Resolved by just saying `AES` instead of AES manifests or AES Charts.